drydock_backups/utils/s3_backups_expiration/main.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
```

Comment:
Is this snippet something that we can include in the infrastructure template?

Reply:
We can include it there if in most cases (installations with AWS) we are going to use the platform bucket to store backups, else we can add a complete tf script to create backups bucket.